payment-instruments/mobile-pay/seamless-view.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Seamless
+redirect_from: /payments/mobile-pay/seamless-view
+estimated_read: 10
```

Comment:
Note: I have not checked the actual time it takes to read through the entire document. This can be done if needed.

Reply:
We have been using https://niram.org/read/, which is a bit moody when it comes to code examples and such. Just add a couple of minutes to account for the code.

Reply:
I used niram and got an estimate of roughly 5 minutes. I added 3 minutes to that for the code, leading to 8 minutes estimated read time.